src/Apps/Order/Routes/Review/index.tsx

```diff
@@ -358,7 +358,20 @@ export const ReviewRoute: FC<ReviewProps> = props => {
     props.router.push(`/orders/${props.order.internalID}/shipping`)
   }
 
-  const { order, isCommittingMutation, isEigen } = props
+  const { order, isCommittingMutation, isEigen, stripe } = props
+  const submittable = !!stripe
+
+  const SubmitButton: FC = () => (
+    <Button
+      variant="primaryBlack"
+      width="100%"
+      loading={isCommittingMutation}
+      disabled={!submittable}
+      onClick={() => onSubmit()}
```

Comment:
This line is the only difference compared to https://github.com/artsy/force/pull/10376. It was incorrectly implemented as

```jsx
onClick={onSubmit}
```

Reply:
This is non-blocking and minor in this case, but it's always good to define components outside of other components at the top level. Have gotten into a lot of subtle trouble here in the past.